project/src/components/video-player/video-player.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useRef, useEffect } from 'react';
+import style from './Video-player.module.css';
+import { FilmType } from '../../types/films-type';
+import Spinner from '../../components/spinner/spinner';
+
+type VideoPlayerProps = {
+  film: FilmType,
+  activeFilm: null | number,
```

Comment:
```suggestion
  activeFilm?: number,
```

если нечего передать передай undefined